### ISSUE TITLE
Images on a front are requested at too low a resolution

### ISSUE THREAD

**Issue by paperboyo** [guardian/frontend#11176]
Hello,

Found another instance of the above. http://www.theguardian.com/uk/film :DVDs and downloads container. The images are shown at 300px wide at 100% zoom but are requested at 220px wide :-(

![low-res-pics-on-culture](https://cloud.githubusercontent.com/assets/6032869/11353669/2740d19c-923d-11e5-81e3-48f25d7e8937.png)

It would be cool to be able to find these automatically, somehow... (as per https://github.com/guardian/frontend/pull/10861#issue-111197150).

Regards
Mateusz

**Comment by OliverJAsh** [comment]
This is happening because “DVDs and downloads” container uses the container type of `fixed/small/slow-IV`, which expects four items when it only has three. E.g. see http://m.code.dev-theguardian.com/olly where I have two containers of the same type, one with three items, one with four items.

I'm not sure if this is a bug or not, or whether this will persist with other container types. My advice for now is switch container type to something that does expect three items.

**Comment by paperboyo** [comment]
Thanks for clarification, @OliverJAsh! I will get a word to the front editor(s) tomorrow. Maybe it’s indeed not something frontend should try to fix. Maybe facia should disallow/warn? (cc/ @piuccio). Or, maybe, nothing, just education, education, education - closing...

**Comment by paperboyo** [comment]
This particular container has now been fixed by CP.

**Comment by paperboyo** [comment]
Just a follow up. So the above was a user’s error. I still think we could prevent it, because it’s gonna happen otherwise (warn in fronts tool or serve bigger images on frontend when the container’s not full). This is it again on yesterday’s uk front:

![three instead of four](https://cloud.githubusercontent.com/assets/6032869/12883521/70dafe8c-ce50-11e5-80ff-9d8c6f13969c.PNG)

But much more interesting is the currently live in brief container on a us front. It uses the same `l-row--cols-4 fc-slice--q-q-q-q` arrangement, BUT there are **four** items there in fronts tool of which only three are showing up (having low res images):

![three instead of four_in brief](https://cloud.githubusercontent.com/assets/6032869/12883597/ddd3d6da-ce50-11e5-9629-27aca5ebd5eb.PNG)

![three instead of four_in brief_fronts config](https://cloud.githubusercontent.com/assets/6032869/12883601/e0cc2d42-ce50-11e5-911c-817eae51e080.PNG)

How so?! cc/ @piuccio

**Comment by piuccio** [comment]
Maybe I'm not following correctly, but this is either one for frontend or not fix is needed.

The collection is configured to have 4 stories, but it's common for collection to have fewer than expected (deduping is the most common cause, MPU another).
The tool doesn't know (and shouldn't know) how many stories are going to be displayed, that's computed downstream.
Only `facia` knows that it's trying to display 3 stories in a 4 columns container.

`facia` logic is smart enough to fallback to a 3 column layout (instead of forcing a 4 columns and leaving one empty), but it feels like that's only half implemented. It seems sensible to me that it should also ask for bigger size images. The size is not fixed by the tool but computed dynamically by facia.

In other words:
- the columns is a function of the number of stories (dynamic)
- the image size is a function of the collection configuration (static)

The latter seems wrong, but I understand there might be technical reasons for that. If you guys can't fix it, there's nothing the tool can do.

@paperboyo education is always a solution, that's why there's a preview button in the tool. Editors should go and have a look.

**Comment by paperboyo** [comment]
Thanks, @piuccio for a thorough explanation. I agree about education, here though, there is no way anybody will spot 220px images instead of 300px while previewing :cry: 

I haven’t reopened, maybe I will hear somebody from frontend...

> It seems sensible to me that it should also ask for bigger size images.

That would be great. But only if it’s not a huge task.
